docs/readme_login.md

```diff
@@ -0,0 +1,304 @@
+# Logging in with PEW EOS Plugin
+
+----------------------------------------------------------------------------------------
+
+## Overview
+
+This file documents how Login works in the PEW EOS Plugin for the following platforms
+* Windows
+* Mac 
+* iOS
+* Android
+
+----------------------------------------------------------------------------------------
+
+## Integration Notes
+
+### Prerequisites
+
+* EOS Platform Interface must be initilaized before performing a login
+
+----------------------------------------------------------------------------------------
+## Login Interfaces
+
+There are two login interfaces.
+* **Auth Login**       for authorizing an **Epic Account** and related services 
+* **Connect Login**    for connecting the **User** for one **ProductID (Game)**
+
+More information about the distinction could be found [here](https://dev.epicgames.com/en-US/news/accessing-eos-game-services-with-the-connect-interface#a-brief-summary-of-auth-vs-connect-interfaces)
+
+### Auth Login
+
+* Auth Login functions are declared in EOS Auth Interface
+* **Account Portal** (`EOS_LCT_AccountPortal`) and **Persistent Auth** (`EOS_LCT_PersistentAuth`) are the primary Auth Types (`LoginCredentialType`) to login  
+  * **Persistent Auth** will login with credentials of the previous successful **Account Portal** login
+* **Dev Auth**(`EOS_LCT_Developer`) is for quick iteration for developers, which could be done by using the Dev-Auth tool provided with the EOS SDK. Find [this](https://github.com/PlayEveryWare/eos_plugin_for_unity/tree/development/docs/Walkthrough.md) for details
```

Comment:
Instead of 'Find this for details' perhaps change it to 'Read this for more details.'.